netZooPy/panda/panda.py

```diff
@@ -448,6 +452,7 @@ def gupdate_diagonal(diagonal_matrix, num, alpha, step):
             # Update motif_matrix
             if computing=='gpu':
                 import cupy as cp
+                motif_matrix=cp.array(motif_matrix)
```

Comment:
Hey @dcolinmorgan, so I noticed in your last PR, that you changed the order in which PANDA calls motif and PPI. To keep the code as originally implemented, I have put reordered the variables in a subsequent PR. Could you please put them back?